requirements.txt

```diff
@@ -1,4 +1,4 @@
-cefpython3==66.1
+cefpython3==66.0

```

Comment:
For some reason I had to roll back to version 66.0 because there was some issue with pypi in the docker base container. I can see in pypi that [cefpython3](https://pypi.org/project/cefpython3/) is currently at v66.1 but during the container build process it would error out saying the latest it could find was 66.0. Just wanted to highlight the reason for this rollback.